keyboards/mntre/config.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2021 Cedric Vincent (@cdc-mkb)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x03EB
+#define PRODUCT_ID   0x2042
```

Comment:
This is already used, and needs to be changed.

https://www.the-sz.com/products/usbid/index.php?v=0x03EB&p=0x2042&n=

Reply:
I reused the same vendor/product IDs as in the original firmware, but I agree with you they must be changed to something less prone to collision.  I checked in QMK and in the USB ID database that the following change should be Ok:
```suggestion
#define VENDOR_ID    0xFEED
#define PRODUCT_ID   0x1302
```